### DISCUSSION TITLE
How to set application name or service name in java application while using collector

### DISCUSSION THREAD

**Discussion by gajavallisantosh** [open-telemetry/opentelemetry-java-instrumentation#10339] [Q&A]
HI, I would like to set application name or service name in my roll dice java application instead of giving service name in OTEL collector. Could you please give me an example regarding that.

**Comment by laurit** [comment]
See https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#opentelemetry-resource
